src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+
+public class Todo extends Task {
```

Comment:
I noticed that you have not done unit testing. Maybe you can introduce new classes to test specific methods?